R/pkg/R/mllib.R

```diff
@@ -694,8 +694,8 @@ setMethod("predict", signature(object = "KMeansModel"),
 #' }
 #' @note spark.mlp since 2.1.0
 setMethod("spark.mlp", signature(data = "SparkDataFrame"),
-          function(data, blockSize = 128, layers = c(3, 5, 2), solver = "l-bfgs", maxIter = 100,
-                   tol = 0.5, stepSize = 1, seed = 1) {
+          function(data, blockSize = 128, layers, solver = "l-bfgs", maxIter = 100,
```

Comment:
If the goal is to require layers to have a value (I didn't realize this from our PR description), then we should have layers as the 2nd parameter (after data) without any default value.

We should also make sure when layers is later coerced to array that its values are coerced into integer?

```
> a <- list(1, 2, "a")
> as.integer(a)
[1]  1  2 NA
Warning message:
NAs introduced by coercion
```